prettier/main.ts

```diff
@@ -107,8 +107,8 @@ async function checkFile(
   const text = await readFileIfExists(filename);
 
   if (!text) {
-    // The file is deleted. Skip.
-    return;
+    // The file is deleted or empty. Skip.
```

Comment:
We changed the way to enumerate files (`git ls-files` -> `fs.wak` in #241) and therefore this function should never be called with deleted files. So I think it's better to remove `deleted or` part from the comment.